toolchain/lc-compile/src/report.c

```diff
@@ -152,6 +169,22 @@ static void _PrintPosition(long p_position)
     fprintf(stderr, "%s:%ld:%ld: ", t_path, t_row, t_column);
 }
 
+/* Print the source code line that contains p_position, and another
+ * line below it with a caret pointing "up" to the exact character
+ * that the position specifies. */
+static void _PrintContext(long p_position)
```

Comment:
@peter-b Could this output either be enabled or skipped by an option. It wouldn't be needed by anything parsing the output and displaying the error itself. Perhaps `-v` would be good...